go/vt/schema/online_ddl.go

```diff
@@ -182,25 +137,186 @@ func ParseOnlineDDLStatement(sql string) (ddlStmt sqlparser.DDLStatement, action
 	return ddlStmt, action, fmt.Errorf("Unsupported query type: %s", sql)
 }
 
+// NewOnlineDDLs takes a single DDL statement, normalizes it (potentially break down into multiple statements), and generates one or more OnlineDDL instances, one for each normalized statement
+func NewOnlineDDLs(keyspace string, ddlStmt sqlparser.DDLStatement, ddlStrategySetting *DDLStrategySetting, requestContext string) (onlineDDLs [](*OnlineDDL), err error) {
+	appendOnlineDDL := func(tableName string, ddlStmt sqlparser.DDLStatement) error {
+		onlineDDL, err := NewOnlineDDL(keyspace, tableName, sqlparser.String(ddlStmt), ddlStrategySetting, requestContext)
+		if err != nil {
+			return err
+		}
+		onlineDDLs = append(onlineDDLs, onlineDDL)
+		return nil
+	}
+	switch ddlStmt := ddlStmt.(type) {
+	case *sqlparser.DropTable:
+		tables := ddlStmt.GetFromTables()
+		for _, table := range tables {
+			ddlStmt.SetFromTables([]sqlparser.TableName{table})
+			if err := appendOnlineDDL(table.Name.String(), ddlStmt); err != nil {
+				return nil, err
+			}
+		}
+		return onlineDDLs, nil
+	case *sqlparser.CreateTable:
+		// handled later on
+	case *sqlparser.AlterTable:
+		// handled later on
+	default:
+		return nil, fmt.Errorf("Unsupported statement for Online DDL: %v", sqlparser.String(ddlStmt))
```

Comment:
nit:`vterrors` is your friend. this applies to quite a few errors in this file.

also, I believe errors should not start with a capitalised letter

Reply:
Replaced all `fmt.Errorf()` in this file with `vterrors`, lower case as needed.